connectors/connectors.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+set -euxo pipefail
+
+VM_CONNECTORS_DIR=/usr/lib/hadoop/lib
+
+declare -A GCS_TO_BQ_VERSIONS_MAP
+GCS_TO_BQ_VERSIONS_MAP=(
+  ["1.7.0"]="0.11.0")
```

Comment:
Is this just to avoid dependency conflicts? Otherwise it seems like "ask for forgiveness rather than permission" is better -- just try to download the version the user specified. If it does not exist, they will get an error anyway. And that way you don't need to update this map for every release.

Reply:
Yes, both connectors have the same classes and they could conflict with each other if connectors are from different releases.